docs/sources/helm-charts/tempo-distributed/get-started-helm-charts/_index.md

```diff
@@ -386,6 +386,15 @@ Update the `storage` configuration options based upon your requirements:
 
 - [Google Cloud Storage configuration documentation](https://grafana.com/docs/tempo/<TEMPO_VERSION>/configuration/hosted-storage/gcs)
 
+#### Azure with the `local_blocks` and metrics-generator processors
```

Comment:
This content is inserted into the instructions for the tempo-distributed Helm charts. We had feedback from a customer they had trouble with the local-blocks processor in the Helm chart. For contents oof the file, refer to shared/azure-metrics-generator.md.